### PR TITLE
fetch an org

### PR DESCRIPTION
**Pull request by mrkev**
Thoughts on having this `AppDevNodeRouter` class to fetch nodes?

### CODE REVIEW

File: src/utils/AppDevNodeRouter.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  constructor() {
+    super("GET");
+  }
+
+  async fetchWithId(id: id): Promise<?T> {
+    throw new Error(`Not implemented for path ${this.getPath()}`)
+  }
+
+  async content(req: Request) {
```
Comment (Jma353) [inline src/utils/AppDevNodeRouter.js]: why is there a space between every line?

File: src/utils/AppDevNodeRouter.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * For fetching nodes.
+ * NOTE: Expects the path to contain an :id field!
+ */
+class AppDevNodeRouter<T> extends AppDevRouter {
+
+  constructor() {
+    super("GET");
+  }
+
+  async fetchWithId(id: id): Promise<?T> {
```
Comment (Jma353) [inline src/utils/AppDevNodeRouter.js]: `// TODO` this since we plan on adding this I suppose?

File: src/utils/AppDevNodeRouter.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+type id = number
+
+/**
+ * For fetching nodes.
+ * NOTE: Expects the path to contain an :id field!
+ */
+class AppDevNodeRouter<T> extends AppDevRouter {
+
+  constructor() {
+    super("GET");
```
Comment (Jma353) [inline src/utils/AppDevNodeRouter.js]: `'GET'`

File: src/utils/AppDevNodeRouter.js (hunk 2 above)
Comment (Jma353) [reply src/utils/AppDevNodeRouter.js]: JK this is an abstract parent my B

**Comment by Jma353** [comment]
I like this - we might wanna make these generic enough to exist in the `appdev.js` module.  We also gotta migrate this backend to use that module.

File: src/utils/AppDevNodeRouter.js (hunk 2 above)
Comment (mrkev) [reply src/utils/AppDevNodeRouter.js]: lol 👍

File: src/utils/AppDevNodeRouter.js (hunk 1 above)
Comment (mrkev) [reply src/utils/AppDevNodeRouter.js]: They are semantically distinct parts

**Pull request merged**